specificatie/genereervariant/openapi.json

```diff
@@ -4338,6 +4338,13 @@
             "items" : {
               "$ref" : "#/components/schemas/HalLink"
             }
+          },
+          "bijbehorendeErfpachters" : {
+            "type" : "array",
+            "description" : "Zakelijk gerechtigden die erfpachter zijn, omdat hun zakelijk recht is ontstaan uit een grondperceel belast met erfpacht.\n",
```

Comment:
de description overal aanvullen met: 
"Dit zijn de erfpachters van het perceel OF de eigenaren van een appartementsrecht op het perceel."

Reply:
@CathyDingemanse wat bedoel je met "overal"? Hier en in _links?

Reply:
Ja, overal waar deze description staat.